pkg/publish.proj

```diff
@@ -14,10 +14,12 @@
 
   <PropertyGroup>
     <PackageDownloadDirectory>$(MSBuildThisFileDirectory)packages\AzureTransfer\</PackageDownloadDirectory>
+    <!-- SignFiles needs IntermediateOutputPath & OutDir to be set -->
+    <IntermediateOutputPath>$(MSBuildThisFileDirectory)bin\obj</IntermediateOutputPath>
```

Comment:
I thought it only needed OutDir but I guess we always had IntermediateOutputPath set in other contexts.